frontend/apps/reader/modules/readerkoptlistener.lua

```diff
@@ -72,6 +72,9 @@ function ReaderKoptListener:onDocLangUpdate(lang)
 end
 
 function ReaderKoptListener:onConfigChange(option_name, option_value)
+    -- font_size is historically and sadly shared by both mupdf and cre reader modules,
+    -- but fortunately they can be distinguished by their different ranges
+    if option_name == "font_size" and option_value > 2 then return end
```

Comment:
So, this `> 2` should be `> 3` if technically, kopt (and may be our fine tuning numberpicker) accepts up to 3.0.
A quick look at crengine and its own m_min_font_size is 6.

(I would personally use `> 5` and `< 5` just to not have to think too much about them :)